README.md

```diff
@@ -76,22 +68,21 @@ Open `http://localhost:3000/` in your browser.
 ### Run tests
 
 ```
-    rpsec --format documentation
+    rspec 
 ```
 
-> Tests will be added by Microverse students. There are no tests for initial features in order to make sure that students write all tests from scratch.
 
 ### Deployment
 
-TBA
+[Live link](https://fast-ravine-48156.herokuapp.com/)
```

Comment:
- [ ] **OPTIONAL** I would suggest you put your Live link in a more visible part so that any interested person can see your deployed app immediately :) like a link in the big image or at the beginning of the page, that is up to you